tests/drop_counters/test_drop_counters.py

```diff
@@ -120,6 +120,82 @@ def enable_counters(duthost):
             logger.info("Restoring counter '{}' state to disable".format(port))
             duthost.command("counterpoll {} disable".format(port))
 
+
+class FanoutConfig(object):
+    def __init__(self, fanout_host, ansible_root, trunk_port, ansible_localhost):
+        self.rule_id = None
+        self.playbook = os.path.join(os.path.dirname(__file__), "exec_template.yml")
+        self.del_rule_template = os.path.join(os.path.dirname(__file__), "mlnx_fanout_del_rule.j2")
+        self.cmd = "cd {ansible_path}; ansible-playbook {playbook} -i lab -l {fanout_host} --extra-vars \"{extra_vars}\" -vvvvv"
+        self.fanout_host = fanout_host
+        self.ansible_root = ansible_root
+        self.trunk_port = trunk_port
+        self.ansible_localhost = ansible_localhost
+
+    def exec_template(self, **kwargs):
+        """
+        Execute Jinja template on fanout switch which creates openflow rule.
+        @kwargs: Contain parameters to pass into 'ansible-playbook' CLI utility.
+        """
+        attempt = 100
+        while attempt > 0:
+            self.rule_id = random.randint(300, 1000)
```

Comment:
range for flow id is [0-65535]. Just use the last possible ID

Reply:
Fixed by using 65535 as default rule ID, if this will be busy script will try to use another one.